aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -191,6 +191,16 @@ module aptos_framework::delegation_pool {
     /// Chaning beneficiaries for operators is not supported.
     const EOPERATOR_BENEFICIARY_CHANGE_NOT_SUPPORTED: u64 = 19;
 
+    /// Commission percentage increase is too large.
+    const ETOO_LARGE_COMMISSION_INCREASE: u64 = 20;
+
+    /// Commission percentage change is too late in this lockup period, and should be done at least 7 days before the lockup cycle ends.
```

Comment:
update comment for 1/4 lockup duration instead of 7 days prior to lockup's end time

Reply:
Good catch! Done.